src/test/java/org/jabref/logic/bst/util/BstTextPrefixerTest.java

```diff
@@ -17,6 +17,13 @@ void prefix() {
         assertPrefix("{\\'e}", "{\\'e}");
         assertPrefix("{\\'{E}}doua", "{\\'{E}}douard Masterly");
         assertPrefix("Ulric", "Ulrich {\\\"{U}}nderwood and Ned {\\~N}et and Paul {\\={P}}ot");
+        assertPrefix("abcd{e}", "abcd{efg}hi");
```

Comment:
Can you rewrite this test class to `@ParameterizedTest`. I think, [`@CsvSource`](https://junit.org/junit5/docs/snapshot/api/org.junit.jupiter.params/org/junit/jupiter/params/provider/CsvSource.html) is enough.

Reply:
ok I'll work on it later.